### PR TITLE
Add ability to set spatial position to a user defined value

### PR DESCRIPTION
**Pull request by mpozy**

### CODE REVIEW

File: test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+              child: TextFormField(
+                  controller: xTextController..text = '0.0',
+                  keyboardType: TextInputType.number,
+                  inputFormatters: [
+                    FilteringTextInputFormatter.allow(RegExp(r'(^\d*\.?\d*)')),
+                  ]),
+            ),
+          )
+        ],
+      ),
```
Comment (graduad) [inline test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart:53]: Could this be a widget? Or do we have a widget like this?

File: test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+          ),
+        ],
+      ),
+      ElevatedButton(
+          onPressed: () {
+            setSpatialPosition(widget.resultDialogContext);
+            Navigator.of(widget.spatialPositiontDialogContext).pop();
+          },
+          style: ElevatedButton.styleFrom(primary: Colors.deepPurple),
+          child: const Text("Set")),
```
Comment (graduad) [inline test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart:98]: Same as the other comment.

File: test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart (hunk 1 above)
Comment (mpozy) [reply test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart:53]: Yes, we have but in Aga's PR. After merge Aga's code I can refactor this.

File: test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart (hunk 2 above)
Comment (mpozy) [reply test_app/lib/screens/participant_screen/spatial_position_dialog_content.dart:98]: Same as above.

**Review by graduad — APPROVED** [review]

**Pull request merged**